crates/log-wasmtime/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+    name    = "log-wasmtime"
+    version = "0.1.0"
+    edition = "2021"
+    authors = [ "Nitish Malhotra <nitishm@microsoft.com>" ]
+
+[lib]
+    doctest = false
+
+[dependencies]
+    anyhow               = "1.0"
+    env_logger = "0.9.0"
+    log = "0.4.0"
+    wit-bindgen-wasmtime = { git = "https://github.com/bytecodealliance/wit-bindgen", rev = "dab589f579b980c81b39c8348400b88a8c4d175a" }
```

Comment:
I think the revision used everywhere else in the repo is `2e654dc82b7f9331719ba617a36ed5967b2aecb0`

Reply:
The current revision points to a version of WIT bindgen that uses Wasmtime v0.32, while the rest of the project has been updated to v0.33.